tests/templates/y_cable_simulator_client.j2

```diff
@@ -16,6 +21,129 @@ helper_logger = logger.Logger(SYSLOG_IDENTIFIER)
 UPPER_TOR = "upper_tor"
 LOWER_TOR = "lower_tor"
 
+PLATFORM_JSON = 'platform.json'
+PORT_CONFIG_INI = 'port_config.ini'
+
+# A dict for mapping physical port to host interface
+g_physical_to_host_port_mapping = {}
+g_physical_to_logical_port_mapping = {}
```

Comment:
This map is not used. What is the purpose of introducing this map?

Reply:
The initial purpose for ```g_physical_to_logical_port_mapping``` is to retrieve interface name from physical port. But the logic is not used yet. The map is already removed.